platform-dom/src/main/resources/com/softicar/platform/dom/elements/dom-button-style.css

```diff
@@ -14,7 +14,7 @@
 
 .DomButton {
 	/* position and layout */
-	justify-content: center;
+	align-items: center;
```

Comment:
semi-related fix; `justify-content` was wrong here before, and caused problems for this PR